doc-content/jbpm-docs/src/main/asciidoc/DesignBuildCases/case-management-comments-con.adoc

```diff
@@ -1,5 +1,5 @@
 = Comments
 
-In case management, comments facilitate collaboration within the case instance, and allow case workers to easily communicate with each other to exchange information.
+In case management, comments facilitate collaboration within the case instance and allow case workers to easily communicate with each other to exchange information.
```

Comment:
Are split infinitives allowed?

Reply:
Where is the split infinitive?

Reply:
"to easily communicate" is a textbook split infinitive. I know it is a debate, so the question is if we have a determined-from-above side in this debate.